src/FSharp.Control.TaskSeq.sln.DotSettings

```diff
@@ -0,0 +1,5 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=infinitum/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=iteri/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=taskseqs/@EntryIndexedValue">True</s:Boolean>
```

Comment:
Adding this one (`taskseqs`) as a 'ignore this for spellchecking' is a bit debatable - taskseqs and asyncseqs are used as shorthand inc in xmldoc, which is pretty cryptic to a newcomer, even if it is a good shorthand that makes sense taking into account that one is trying to convey a relatively difficult concept without assuming any given background

Reply:
Not sure what you want to change here, you added this file to the PR ;). Ill merge, we can always fix it later.

Reply:
Yeah, merge on
But on reflection I should not have added that so will try to remember to remove that line in future